code/obj/item/clothing/suits.dm

```diff
@@ -2067,6 +2067,15 @@ TYPEINFO(/obj/item/clothing/suit/space/industrial/salvager)
 		src.AddComponent(/datum/component/wearertargeting/energy_shield, list(SLOT_WEAR_SUIT), 0.8, 1, FALSE, 2)
 		src.AddComponent(/datum/component/power_cell, 100, 100, 5, 30 SECONDS, TRUE)
 
+/obj/item/clothing/suit/security_badge/paper
+	name = "Hall Monitor Badge"
+	desc = "A piece of soggy notebook paper with a red S doodled on it, presumably to represent security."
```

Comment:
Badge owner/job is only ever set for jobs that have receive_badge in jobprocs and never called anywhere else, considering you are spawning it on them and this could do with new logic on new() to set badge_owner_name and badge_owner_job to whoever we spawned on